### PR TITLE
Update doc references in menu and integrations

### PR DESCRIPTION
**Pull request by dennyglee**
Updates the menu bar to refer to the correct docs (docs.delta.io).  The integrations includes the docs and references  sections so each connector contains their associated docs and references.

### CODE REVIEW

**Comment by dennyglee** [comment]
Hey @tdas - please review this version of the docs/integration - thanks!

File: src/data/connectors.json [hunk 1]
```diff
@@ -1,121 +1,160 @@
...
   },
-  {
-    "name": "Trino/Delta Reader (Preview)",
-    "description": "This connector leverages the Delta Standalone project for Trino to read from Delta Lake tables.",
-    "url": "https://github.com/vkorukanti/trino/tree/359-delta",
-    "tags": ["Trino"],
-    "thumbnail": "./images/connectors/trino-logo.png"
-  },
   {
     "name": "PrestoDB/Delta Reader",
```
Comment (tdas) [inline src/data/connectors.json:21]: lets just call it connector as well

File: src/data/connectors.json [hunk 2]
```diff
@@ -1,121 +1,160 @@
...
+    "description": "Apache Spark™ natively reads from and writes to Delta Lake with this package.",
+    "docs": "https://docs.delta.io",
+    "reference": "https://docs.delta.io/latest/delta-apidoc.html",
+    "url": "https://docs.delta.io",
+    "tags": ["Spark"],
+    "thumbnail": "./images/connectors/apache-spark-icon.png"
   },
   {
-    "name": "Flink/Delta Sink",
+    "name": "Flink/Delta Sink (Preview)",
```
Comment (tdas) [inline src/data/connectors.json:12]: Lets just call it Apache Flink connector. no need to change this when we will have source and stuff

File: src/components/CardDataList.jsx [hunk 3]
```diff
@@ -139,6 +147,9 @@ CardDataList.propTypes = {
         }),
       ]).isRequired,
       description: string.isRequired,
+      docs: string,
+      reference: string,
+      version: string,
```
Comment (tdas) [inline src/components/CardDataList.jsx:152]: Whats the version for? i dont see it being used

File: src/data/connectors.json [hunk 4]
```diff
@@ -1,121 +1,160 @@
...
-  {
-    "name": "Trino/Delta Reader (Preview)",
-    "description": "This connector leverages the Delta Standalone project for Trino to read from Delta Lake tables.",
-    "url": "https://github.com/vkorukanti/trino/tree/359-delta",
-    "tags": ["Trino"],
-    "thumbnail": "./images/connectors/trino-logo.png"
-  },
   {
     "name": "PrestoDB/Delta Reader",
     "description": "This connector leverages the Delta Standalone project for Trino to read from Delta Lake tables.",
```
Comment (tdas) [inline src/data/connectors.json:22]: update description to prestoDB

Comment (tdas) [inline src/data/connectors.json:22]: also.. the descriptions are all over the place.. can they be made consistent in structure? some of them say natively .. and others dont.. which raises questions. right now every one reads/writes natively.

File: src/data/connectors.json [hunk 5]
```diff
@@ -1,121 +1,160 @@
...
+  {
+    "name": "Athena Query Federation/Delta Reader (Preview)",
+    "description": "This connector leverages the Delta Standalone project for Apache Beam to read from Delta Lake tables.",
+    "url": "https://github.com/awslabs/aws-athena-query-federation/pull/509",
+    "tags": ["Athena Query Federation"],
+    "thumbnail": "./images/connectors/AthenaFederation1.png"
+  },
+  {
+    "name": "Power BI Delta Connector",
+    "description": "Reading Delta Lake tables natively in Power BI The provided PowerQuery/M function allows you to read a Delta Lake table directly from any storage supported by PowerBI. Most common storage would be Azure Data Lake Store, Azure Blob Storage, or a local folder or file share.",
```
Comment (tdas) [inline src/data/connectors.json:110]:
```suggestion
    "description": "Reading Delta Lake tables natively in Power BI. The provided PowerQuery/M function allows you to read a Delta Lake table directly from any storage supported by PowerBI. ",
```

File: src/data/connectors.json [hunk 6]
```diff
@@ -1,121 +1,160 @@
...
+    "url": "https://github.com/delta-io/connectors/tree/master/powerbi",
+    "tags": ["PowerBI"],
+    "thumbnail": "./images/connectors/powerbi.png"
+  },
+  {
+    "name": "Power BI Delta Sharing Connector (beta)",
+    "description": "Reading Delta Sharing tables natively in Power BI.",
```
Comment (tdas) [inline src/data/connectors.json:117]:
```suggestion
    "description": "Reading Delta tables in Power BI using the Delta Sharing.",
```

File: src/data/connectors.json (hunk 1 above)
Comment (dennyglee) [reply src/data/connectors.json]: Fixed, thanks!

File: src/data/connectors.json (hunk 2 above)
Comment (dennyglee) [reply src/data/connectors.json]: Calling it Apache Flink now.

File: src/data/connectors.json [hunk 7]
```diff
@@ -1,121 +1,160 @@
...
+    "name": "dlt - Delta Lake interface for SparkR",
+    "description": "This package provide readers and writers for the Delta format, DeltaTable merge API, and Delta table builder API for SparkR.",
+    "docs": "https://dlt.zero323.net/",
+    "reference": "https://dlt.zero323.net/reference/index.html",
+    "url": "https://dlt.zero323.net/",
+    "tags": ["SparkR"],
+    "thumbnail": "./images/connectors/dlt.png"
+  },
+  {
+    "name": "Beam/Delta Reader (Preview)",
```
[the remark this answers is not in the document]
Comment (dennyglee) [reply src/data/connectors.json]: Ack

File: src/data/connectors.json (hunk 4 above)
Comment (dennyglee) [reply src/data/connectors.json]: ack!

Comment (dennyglee) [reply src/data/connectors.json]: Yes, language is now consistent

File: src/data/connectors.json [hunk 8]
```diff
@@ -1,121 +1,160 @@
...
+    "description": "This connector leverages the Delta Standalone project for Apache Beam to read from Delta Lake tables.",
+    "url": "https://github.com/mbenenso/beam-deltalake",
+    "docs": "https://github.com/mbenenso/beam-deltalake",
+    "reference": "https://github.com/mbenenso/beam-deltalake-example",
+    "tags": ["Beam"],
+    "thumbnail": "./images/connectors/apache-beam.png"
+  },
+  {
+    "name": "Athena Query Federation/Delta Reader (Preview)",
+    "description": "This connector leverages the Delta Standalone project for Apache Beam to read from Delta Lake tables.",
```
[the remark this answers is not in the document]
Comment (dennyglee) [reply src/data/connectors.json]: Fixed

File: src/data/connectors.json (hunk 5 above)
Comment (dennyglee) [reply src/data/connectors.json]: New shorter description

File: src/data/connectors.json (hunk 6 above)
Comment (dennyglee) [reply src/data/connectors.json]: New shorter description

File: src/components/CardDataList.jsx (hunk 3 above)
Comment (dennyglee) [reply src/components/CardDataList.jsx]: Version and reference have been removed and replaced with source code per our chat.

**Comment by dennyglee** [comment]
@tdas Please review when you get the chance.  Thanks!

File: src/data/connectors.json [hunk 9]
```diff
@@ -1,121 +1,165 @@
...
   },
+
   {
     "name": "Athena",
-    "description": "Athena supports reading from external tables using a manifest file, which is a text file containing the list of data files to read for querying a table. When an external table is defined in the Hive metastore using manifest files, Athena can use the list of files in the manifest rather than finding the files by directory listing.",
+    "description": "This utility allows Athena to read from Delta Lake using a manifest file.",
     "url": "https://docs.delta.io/latest/presto-integration.html",
-    "tags": ["Athena", "AWS"],
+    "docs": "https://docs.delta.io/latest/presto-integration.html",
+    "source_code": "https://docs.delta.io/latest/presto-integration.html",
```
Comment (tdas) [inline src/data/connectors.json:143]: same as above, no need for source code link

File: src/data/connectors.json [hunk 10]
```diff
@@ -1,121 +1,165 @@
...
     "thumbnail": "./images/connectors/sql.png"
   },
   {
     "name": "Kafka Delta Ingest",
-    "description": "The kafka-delta-ingest project aims to build a highly efficient daemon for streaming data through Apache Kafka into Delta Lake. This project is currently highly experimental and evolving in tandem with the delta-rs bindings.",
+    "description": "This project builds a highly efficient daemon for streaming data through Apache Kafka into Delta Lake.",
     "url": "https://github.com/delta-io/kafka-delta-ingest",
-    "tags": ["Kafka"],
+    "docs": "https://github.com/delta-io/kafka-delta-ingest/blob/main/doc/DESIGN.md",
+    "source_code": "https://github.com/delta-io/kafka-delta-ingest/tree/main/src",
```
Comment (tdas) [inline src/data/connectors.json:79]:
```suggestion
    "source_code": "https://github.com/delta-io/kafka-delta-ingest/",
```

File: src/data/connectors.json [hunk 11]
```diff
@@ -1,121 +1,165 @@
...
+    "source_code": "https://github.com/delta-io/connectors/tree/master/powerbi",
+    "tags": ["PowerBI"],
+    "thumbnail": "./images/connectors/powerbi.png"
+  },
+  {
+    "name": "Power BI | Delta Sharing Connector (Beta)",
+    "description": "This connector allows Power BI to read from Delta Sharing endpoint.",
+    "url": "https://docs.microsoft.com/en-us/power-query/connectors/deltasharing",
+    "docs": "https://docs.microsoft.com/en-us/power-query/connectors/deltasharing",
+    "source_code": "https://docs.microsoft.com/en-us/power-query/connectors/deltasharing",
```
[the remark this answers is not in the document]
Comment (dennyglee) [reply src/data/connectors.json]: fixed

File: src/data/connectors.json (hunk 10 above)
Comment (dennyglee) [reply src/data/connectors.json]: fixed

**Pull request merged**